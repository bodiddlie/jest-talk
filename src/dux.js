export const ACTION_ONE = 'ACTION_ONE';
export const ACTION_TWO = 'ACTION_TWO';

export const doActionOne = () => {
  return {type: ACTION_ONE};
}

export const doActionTwo = (id) => {
  return {type: ACTION_TWO, id};
}