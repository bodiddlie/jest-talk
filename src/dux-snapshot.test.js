import {doActionOne, doActionTwo} from './dux';

describe('Snapshot Action Creators', () => {
  test('creates an action one type', () => {
    expect(doActionOne()).toMatchSnapshot();
  });

  test('creates an action two type', () => {
    expect(doActionTwo(1)).toMatchSnapshot();
  });
});