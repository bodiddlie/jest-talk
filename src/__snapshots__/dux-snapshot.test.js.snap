exports[`Snapshot Action Creators creates an action one type 1`] = `
Object {
  "type": "ACTION_ONE",
}
`;

exports[`Snapshot Action Creators creates an action two type 1`] = `
Object {
  "id": 1,
  "type": "ACTION_TWO",
}
`;
